Source/Core/Common/FileUtil.cpp

```diff
@@ -149,13 +149,13 @@ bool CreateDir(const std::string& path)
 #ifdef _WIN32
   if (::CreateDirectory(UTF8ToTStr(path).c_str(), nullptr))
     return true;
-  DWORD error = GetLastError();
-  if (error == ERROR_ALREADY_EXISTS)
+  if (GetLastError() == ERROR_ALREADY_EXISTS)
   {
     WARN_LOG(COMMON, "CreateDir: CreateDirectory failed on %s: already exists", path.c_str());
     return true;
   }
-  ERROR_LOG(COMMON, "CreateDir: CreateDirectory failed on %s: %i", path.c_str(), error);
+  ERROR_LOG(COMMON, "CreateDir: CreateDirectory failed on %s: %s", path.c_str(),
+            GetLastErrorMsg().c_str());
```

Comment:
Generally you should not call GetLastError more than once for a given cause (even if you know _currently_ the code cannot do something to set the last error to something else in between).